### PR TITLE
Add new CSP node IcmpTypesAndCodes

### PR DESCRIPTION
**Pull request by jess-krynitsky**
We have added support for a new firewall rule setting and corresponding CSP node allowing users to specify ICMP types and codes. I have updated the tree and description on this page to reflect the new settings.

### CODE REVIEW

**Comment by jess-krynitsky** [comment]
Adding author @Dansimp to review. Thanks!

File: windows/client-management/mdm/firewall-csp.md [hunk 1]
```diff
@@ -340,6 +341,13 @@ Comma separated list of ranges, For example, 100-120,200,300-320.
 If not specified, the default is All.
 Value type is string. Supported operations are Add, Get, Replace, and Delete.
 
+
+<a href="" id="icmptypesandcodes"></a>**FirewallRules/_FirewallRuleName_/IcmpTypesAndCodes**
+ICMP types and codes applicable to the firewall rule. To specify all ICMP types and codes, use the “*” character. For specific ICMP types and codes, use the “:” character to separate the type and code. For example, 3:4, 1:*. The “*” character can be used to represent any code. The “*” character cannot be used to specify any type, examples such as “*:4” or “*:*” are invalid.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/firewall-csp.md:346]:
```suggestion
ICMP types and codes applicable to the firewall rule. To specify all ICMP types and codes, use the “/*” character. For specific ICMP types and codes, use the “:” character to separate the type and code, for example, 3:4, 1:/*. The “/*” character can be used to represent any code. The “/*” character cannot be used to specify any type; examples such as “/*:4” or “/*:/*” are invalid.
```

File: windows/client-management/mdm/firewall-csp.md [hunk 2]
```diff
@@ -340,6 +341,13 @@ Comma separated list of ranges, For example, 100-120,200,300-320.
...
+If not specified, the default is All. 
+Value type is string. Supported operations are Add, Get, Replace, and Delete.
+
+
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/firewall-csp.md:350]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @jess-krynitsky, kindly commit the suggested changes. The "\" is needed for the text to render correctly (please see the preview). Many thanks.

**Review by jess-krynitsky — COMMENTED** [review]
Committing suggested changes for proper rendering. However, the "*" character is not showing up in the preview now so I will need to check in an additional change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by jess-krynitsky** [comment]
@JohanFreelancer9 thanks for the PR approval. What need to be done to merge this change with the base branch?

**Review by aczechowski — APPROVED** [review]

**Pull request merged**